src/lib/dialog/dialog-container.ts

```diff
@@ -95,6 +98,7 @@ export class MatDialogContainer extends BasePortalOutlet {
     private _elementRef: ElementRef,
     private _focusTrapFactory: FocusTrapFactory,
     private _changeDetectorRef: ChangeDetectorRef,
+    private _focusMonitor: FocusMonitor,
```

Comment:
@jelbourn should we make this optional? It's technically a breaking signature change, but the dialog container is an internal component.

Reply:
I think this one should be okay